contrib/cirrus/test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+source $(dirname $0)/lib.sh
+
+[[ -x "bin/oci-seccomp-bpf-hook" ]] || \
+    make
+
+echo "Installing oci-seccomp-bpf-hook"
+make install PREFIX=/usr
+
+echo "Executing integration tests"
+make test-integration BATS_OPTS=--tap
```

Comment:
Those need to be executed as `root`. Do we need to add a `sudo` here?

Reply:
Nope, we're root by default.  Though the bats test output is not helpful...I'm making a wild-guess the failure has something to do with podman-1.5 and conmon.  Meaning we have ***both*** `/usr/libexec/podman/conmon` *and* `/usr/bin/conmon`.

Reply:
I think we can wait a little for F30 to pick up `podman 1.6.somerthing` and see if that fixes things.